File: lambda functions/Get_Watch_Next_by_Id/Next.js
const mongoose = require('mongoose');

const watch_next_schema = new mongoose.Schema({
    _id: String,
    url: String,
    title: String,
    speakers: String,
    publishedAt: String,
});

const talk_schema = new mongoose.Schema({
    _id: String,
    watch_next: [watch_next_schema]
}, { collection: 'tedx_data' });
module.exports = mongoose.model('talk', talk_schema);

File: mytedx/mytedx/lib/models/next.dart
class Next {
  final String id;
  final String url;
  final String title;
  final String mainSpeaker;
  final String publishedAt;

  Next.fromJSON(Map<String, dynamic> jsonMap) :
    id=jsonMap['_id'],
    url=jsonMap['url'],
    title = jsonMap['title'],
    mainSpeaker = (jsonMap['speakers'] ?? ""),
    publishedAt = (jsonMap['url'] ?? "");
}